src/Handlebars/Context.php

```diff
@@ -105,6 +127,16 @@ public function popIndex()
         return array_pop($this->index);
     }
 
+    /**
+     * Pop the last section data from the stack.
+     *
+     * @return array Last data
+     */
+    public function popData()
+    {
+        return array_pop($this->dataStack);
```

Comment:
Similar comment here regarding the options and exception.

Reply:
Agreed and fixed.